master/usage/configuration/as-service.md

```diff
@@ -32,36 +32,36 @@ CALICO_NETWORKING_BACKEND=bird
 Be sure to update this environment file as necessary, such as modifying
 ETCD_ENDPOINTS to point at the correct etcd cluster endpoints.
 
-> **NOTE**
->
-> The ETCD_CA_FILE, ETCD_CERT_FILE, and ETCD_KEY_FILE
-> environment variables are required when using Etcd with SSL/TLS.  The values
-> here are standard values for a non-SSL version of Etcd, but you can use this
+> **Note**: The `ETCD_CA_FILE`, `ETCD_CERT_FILE`, and `ETCD_KEY_FILE`
+> environment variables are required when using etcd with SSL/TLS. The values
+> here are standard values for a non-SSL version of etcd, but you can use this
 > template to define your SSL values if desired.
 >
-> If CALICO_NODENAME is blank, the compute server hostname will be used
+> If `CALICO_NODENAME` is blank, the compute server hostname will be used
 > to identify the Calico node.
 >
-> If CALICO_IP or CALICO_IP6 are left blank, Calico will use the currently
-> configured values for the next hop IP addresses for this node - these can
+> If `CALICO_IP` or `CALICO_IP6` are left blank, Calico will use the currently
+> configured values for the next hop IP addresses for this node—these can
```

Comment:
Out of interest: is this changed character an em-dash?  If so, do you know if it also works to type it as `---` (which some Internet hits say).  There's possibly a tiny worry here about whether non-ASCII characters like this will display correctly and by easily typable in everyone's editor.  (Although it really should be - we've been in a Unicode world for years now!)

Reply:
Good question! This is something that is a little different in the GitHub flavored Markdown syntax, where `---` does not become the proper em dash. Other flavors *do* do this as I notice you (and others) would expect. However, it is easy enough to work around this, you can simply insert the actual em dash, as I've done here. (You can also include the HTML character code, but it's not as readable as you will likely agree).